tests/serialization/test_dag_serialization.py

```diff
@@ -1093,14 +1093,12 @@ def test_no_new_fields_added_to_base_operator(self):
         tests should be added for it.
         """
         base_operator = BaseOperator(task_id="10")
-        fields = base_operator.__dict__
+        fields = {k: v for (k, v) in vars(base_operator).items() if k in BaseOperator.get_serialized_fields()}
```

Comment:
Do we have a test to ensure `get_serialized_fields()` is up-to-date? Otherwise we may accidentally add a field on BaseOperator but forget to serialize it, and this test would fail to reflect that.

Reply:
We don't, no, this is it. This will pick up anything added via `self.x` in the constructor, but won't pick up things like `x: int = 0` at the class level.

Think it's worth it?